internal/pkg/agent/application/actions/handlers/handler_action_settings.go

```diff
@@ -53,6 +50,12 @@ func (h *Settings) Handle(ctx context.Context, a fleetapi.Action, acker acker.Ac
 		return fmt.Errorf("invalid log level, expected debug|info|warning|error and received '%s'", action.LogLevel)
 	}
 
+	lvl := logp.InfoLevel
```

Comment:
[Blockerish]
Would you mind adding a type alias on `github.com/elastic/elastic-agent/pkg/core/logger` for the log level? It'd be better not leak this as none of these files were importing `logp`. Besides it's passing a `logp.Level` to a seemingly unrelated package, it's confusing at first glace.

Reply:
`logp.Level` is used through out the entire code base. I agree it might be nice to make this change, but doing in a PR for something that needs to be fixed in 8.6 is not the correct place. Currently the `logp.` is referenced 72 times across 31 different files in the elastic-agent repository.

Reply:
I agree with Blake on this, we can do this as a follow up if we feel it is important enough to address. It shouldn't block this PR, it isn't a bug or flaw in the implementation.

Reply:
ok, makes sense :)